scripts/testdata/satellite-config.yaml.lock

```diff
@@ -118,6 +118,12 @@ contact.external-address: ""
 # the time between each send of garbage collection filters to storage nodes
 # garbage-collection.interval: 120h0m0s
 
+# size of the buffer used to batch inserts into the transfer queue.
+# graceful-exit.chore-batch-size: 500
+
+# how often to run the transfer queue chore.
+# graceful-exit.chore-interval: 30s
```

Comment:
30s seems too frequent to me. I think we should start a discussion in #config-changes, but I feel like anywhere from 1h to 1d would be an ideal interval for this process.

Reply:
leaving as is based on the changes below